book.json

```diff
@@ -1,6 +1,9 @@
 {
     "gitbook": ">=3.2.0",
     "variables": {
+      "py_version": "3.9",
+      "py_release": "3.9.0",
+      "pyaw_version": "3.8",
```

Comment:
As this isn't the version of PythonAnywhere, but of Python **on** PythonAnywhere, maybe abbreviate it as `pa_py_version` or `py_version_pa`. Or spell "PythonAnywhere" out completely: `py_version_on_pythonanywhere`

Reply:
@das-g pushed commit: https://github.com/DjangoGirls/tutorial/pull/1683/commits/bebcedb20dad37022c6ce0adcf41394df71035c9